allennlp/tests/training/metrics/wikitables_accuracy_test.py

```diff
@@ -29,7 +31,8 @@ def test_accuracy_is_scored_correctly(self):
         logical_form = ('((reverse fb:row.row.year) (fb:row.row.index (max '
                         '((reverse fb:row.row.index) (fb:row.row.league fb:cell.usl_a_league)))))')
 
-        wikitables_accuracy = WikiTablesAccuracy(table_directory='tests/fixtures/data/wikitables/')
+        print(os.getcwd())
```

Comment:
@nelson-liu is this an intentional `print`?

Reply:
it's not, thanks for catching that!